src/test/coins_tests.cpp

```diff
@@ -562,6 +564,15 @@ BOOST_AUTO_TEST_CASE(ccoins_serialization)
     }
 }
 
+struct CoinStruct {
+    const CAmount value;
+    const char flags;
```

Comment:
I still think this should be an enum named `EntryState` or something, with all combinations of flags like https://github.com/bitcoin/bitcoin/pull/30906/commits/ba52ebd41673abf63d4495215c84be7244469d30#r1760349038. That's the proper way to do this instead of defining static vars for each value of a char we care about.

Reply:
Absolutely, that's ny plan for tomorrow - but needed to clean up the test before doing that